README.md

```diff
@@ -193,6 +197,20 @@ This is how you actually render the React components you exposed to `window` ins
 
 This is a helper method that takes any JavaScript expression and returns the output from evaluating it. If you have more than one line that needs to be executed, wrap it in an IIFE. JS exceptions will be caught and console messages handled properly.
 
+### Redux
+See examples of using a generator function to create the top level react-redux component.
+
+### React-Router
+There's no special instructions needed for react-router for client rendering.
+
+If you're server rendering with react-router, you'll need to use the option generator_function to return either:
```

Comment:
And it's not the generator function that is returning these things, right. You are assigning one of these two things to the property you are feeding to `generator_function`.